src/main/java/frc/robot/subsystems/drive/SwerveModule.java

```diff
@@ -66,6 +67,8 @@ public SwerveModule(double x,                   // (positive right) relative to
         _rotatePID.setOutputRange(-1, 1);
 
         _rotatePID.setSetpointDeadband(2.5);// 5 degree deadband recommended online by other teams
+
+        RobotLog.getInstance().log("Created Swerve Module");
```

Comment:
This log statement isn't necessary. We'll know our swerve modules are created at the same time the drive subsystem as a whole is created.

We can remove this log statement and the import for RobotLog